autofit/__init__.py

```diff
@@ -33,15 +34,14 @@
 from .optimize.grid_search import GridSearch as OptimizerGridSearch
 from .optimize import *
 from .optimize.non_linear.downhill_simplex import DownhillSimplex
-from .optimize.non_linear.downhill_simplex import DownhillSimplex
```

Comment:
I actually think we can probably clean up these imports using from .module import * pattern

Reply:
For Euclid, I am going to have to rework imports a bit anyway, so will sort next week.